assets/minecraft/models/item/diamond_sword.json

```diff
@@ -21,7 +21,7 @@
 		{ "predicate": {"custom_model_data": 15}, "model": "item/mob/rohana/rohana_walking"},
 		{ "predicate": {"custom_model_data": 16}, "model": "item/mob/rohana/rohana_hit"},
 		{ "predicate": {"custom_model_data": 17}, "model": "item/mob/tamaugachi/tamaugachi"},
-		{ "predicate": {"custom_model_data": 18}, "model": "item/mob/tamaugachi/tamaugachi"},
+		{ "predicate": {"custom_model_data": 18}, "model": "item/mob/tamaugachi/tamaugachi_move"},

```

Comment:
I don't see this in the rp

Reply:
its in lightnings PR, the one for hiso etc which you have "required changes for (should also have solved those).
The foodstuff is just removing old ones that were never used, duplicates or just wrongly named

Reply:
Alright then